tests/test_cms.py

```diff
@@ -457,7 +457,10 @@ def test_internal_user_can_not_see_publish_unpublish_buttons_on_edit_page(
 def test_order_measures_in_subtopic(app, db, db_session, test_app_client, mock_rdu_user, stub_subtopic_page):
     guids = [0, 1, 2, 3, 4]
     for guid in guids:
-        stub_subtopic_page.children.append(MeasureVersion(id=guid, guid=str(guid), version="1.0", position=guid))
+        db.session.add(Measure(id=guid, position=guid, slug=str(guid)))
```

Comment:
Might be neater to also include the stub_measure_page fixture to avoid adding the measure here?  But this is probably fine as it is.

Reply:
Each measure version we're creating is in a different measure, so I don't think stubs work. Also not too worried about this intermediary code being super perfectly clean. I think it's OK.